AT 03.01.09 | Multiconfiguration Project > Configure > add description

bahasadyr opened the pull request:
https://trello.com/c/3FY8lhJC/569-tc-030109-multi-configuration-project-configure-add-description

bahasadyr commented:
AT 03.06.09 | Multiconfiguration Project > Configure

olpolezhaeva commented:
В заголовке нужно изменить на AT 03.06.09
После Configure у вас нет сути, что тест делает. В Тест-кейсе и в карточке AT вам это нужно дописать в заголовке.

bahasadyr commented:
> 

Added AT to the headers. Refactored TC according reviewer's comments. Closed PR which was opened a week ago

olpolezhaeva commented:
@bahasadyr Ок! А теперь почему карточка AT 03.06.09 | Multiconfiguration Project > Configure не содержит add description?
И почему она находится в колонке backlog?

olpolezhaeva reviewed (changes requested):
Приведите, пожалуйста, карточки в Trello в порядок и дождитесь review на TC

bahasadyr commented:
> @bahasadyr Ок! А теперь почему карточка AT 03.06.09 | Multiconfiguration Project > Configure не содержит add description? И почему она находится в колонке backlog, не имеет вложений TC и US и вы к ней не присоединились? Получено ли review на TC?

I Added description to AT and TC 03.01.09. Added PR link to AT card. Added TC and US connection to AT. Sorry, I dont have russian keyboard here, only on the phone (

bahasadyr commented:
> Приведите, пожалуйста, карточки в Trello в порядок и дождитесь review на TC

refactored cards in Trello , got approval for this TC : 
https://trello.com/c/3FY8lhJC/569-tc-030109-multi-configuration-project-configure-add-description

olpolezhaeva commented:
Заголовок этого пулл реквеста все еще отличается т карточки.AT 03.01.09 или AT 03.06.09?

bahasadyr commented:
> Заголовок этого пулл реквеста все еще отличается т карточки.AT 03.01.09 или AT 03.06.09?

@olpolezhaeva sorry for the confusion, but AT 03.06.09 is for different TC (https://trello.com/c/JHkh7LJZ/208-tc-030602-multiconfiguration-project-configure) which I havent started to work on it  yet

veronmolosh reviewed (approved):
Looks good!
Maybe I little changed this 
getDriver().findElement(By.xpath("**//textarea[@name='description']"**)).sendKeys(multiConfProjectDescriptionText);
on 
getDriver().findElement(By.name("description")).sendKeys(multiConfProjectDescriptionText);
If it does not interfere with the locator, but this is a matter of taste : )

bahasadyr commented:
> Looks good! Maybe I little changed this getDriver().findElement(By.xpath("**//textarea[@name='description']"**)).sendKeys(multiConfProjectDescriptionText); on getDriver().findElement(By.name("description")).sendKeys(multiConfProjectDescriptionText); If it does not interfere with the locator, but this is a matter of taste : )

@veronmolosh there was another  locator with description on this page ,as I remember
